src/app/find/locationDetails/LocationDetails.jsx

```diff
@@ -280,6 +280,7 @@ class LocationDetails extends Component {
           className="locationCloseButton"
           name="times"
           onClick={goBack}
+          style={{ marginRight: '10px' }}
```

Comment:
The styles for `locationCloseButton` are in locationDetails.css.

Reply:
And btw, I figure increasing the `right` value there might have the same effect as adding `margin-right`.